docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+# Managing relations through the REST API
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through REST requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API. The `connect`, `disconnect`, and `set` attributes, passed in the body of POST or PUT requests, have different behaviors:
+
+- `connect` and `disconnect` perform partial updates
+- `set` performs a full update, effectively replacing all existing relations
+
+```json
```

Comment:
Agree, maybe giving an example of the request as you said, or putting a note/link of the example of Update & Create entry could improve it. 

Linking both examples I think would give more context of what this relation management is about and how you can use it.

But giving context with a request example would be clearer and consistent with the REST API section :).
Would it be useful to transform this code block to tabs, and show one request example per each type?


> Also, maybe it would be interesting to specify what is the behavior when it is applied on a x-to-one relation (poke @Marc-Roig )

Yes, we never mention the difference between managing different types of relations, and maybe we should also.

- This syntax is specially made for one-to-many, many-to-many and many-ways relationships 
- For one-to-one, many-to-one  and one-way the syntax is supported but:
	- Only the last relation on the connect/set will be used.
	- The disconnect array will be ignored.
	- You can also use this format for this ones:
```js
{
  data: {
    category: 2
  }
}

```

I think we should make the distinction somehow in this document. Wdyt @pwizla @petersg83 ?

Reply:
I agree!
Is the disconnect ignored all the time or only when there is also connect?

Reply:
I checked. You are right, it is not ignored for one-to-one & many-to-one. 
When using the connect is not ignored but the connect array overrides anything that was previously in the database.

Reply:
Thanks a lot for the recommendation, guys!